spring-cloud-dataflow-shell/src/main/java/org/springframework/cloud/dataflow/shell/command/RuntimeCommands.java

```diff
@@ -57,9 +57,13 @@
 public class RuntimeCommands implements CommandMarker {
 
 	private static final String LIST_MODULES = "runtime modules";
+	
+	private DataFlowShell dataFlowShell;
 
 	@Autowired
-	private DataFlowShell dataFlowShell;
+	public void setDataFlowShell(DataFlowShell dataFlowShell) {
```

Comment:
I believe this is needed only for testing purpose where we use mocks and the `DataFlowShell` isn't available in the context already. I think we would need to `autowire` the `DataFlowShell` anyway as all the commands are scanned as components, is it not the case?